x-pack/platform/plugins/shared/stack_connectors/common/auth/oauth2_delete_token_axios_interceptor.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ConnectorTokenClientContract } from '@kbn/actions-plugin/server/types';
+import type { AxiosError, AxiosResponse } from 'axios';
+
+interface GetOauth2DeleteTokenAxiosInterceptorParams {
+  connectorTokenClient: ConnectorTokenClientContract;
+  connectorId: string;
+}
+export const getOauth2DeleteTokenAxiosInterceptor = ({
```

Comment:
tested as integration in x-pack/platform/plugins/shared/stack_connectors/server/connector_types/webhook/get_axios_config.test.ts